openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx

```diff
@@ -121,11 +121,15 @@ const DropDownList: FunctionComponent<DropDownListProp> = ({
         {item.type === 'user' ? (
           <UserTag id={item.value as string} name={item.name as string} />
         ) : (
-          <p
-            className={classNames('tw-truncate', widthClass)}
-            title={item.name as string}>
-            {item.name}
-          </p>
+          <>
+            {item.icon}
+            <span className="tw-p-1" />
```

Comment:
Can we take a different approach here?

Reply:
Done. Also made similar changes for Users component dropdown.